include/globals.h

```diff
@@ -710,9 +710,13 @@ extern RemoteDebug Debug;           // Let everyone in the project know about it
     #define PROJECT_NAME            "Ledstrip"
     #endif
 
+    #ifndef ENABLE_WEBSERVER
     #define ENABLE_WEBSERVER            0   // Turn on the internal webserver
+    #endif
```

Comment:
This allows ENABLE_WEBSERVER to be set from the command-line for this project, which is useful for certain debugging scenarios.